src/components/ShowQuestionComponent.tsx

```diff
@@ -78,7 +79,7 @@ export const ViewQuestionComponent = ({
           {players.map((pl, ind) => {
             return (
               <Radio
-                key={ind}
+                key={uuidv4()}
```

Comment:
This will just generate a new uuid each time you render the radio button, effectively forcing a re-render always.
You don't need uuid for this, you can just pass `Math.random()`. The point of a uuid is to generate UNIQUE IDs that you store in the question, not generate a new one each time